mopidy/http/actor.py

```diff
@@ -1,3 +1,4 @@
+import secrets
```

Comment:
This should be sorted between `logging` and `threading`. I'm a bit surprised flake8 didn't catch that for you.